### ISSUE TITLE
generated tarball uses #!/usr/bin/sh while it should use #!/bin/sh

### ISSUE THREAD

**Issue by maxice8** [libexif/libexif#44]

**Comment by dfandrich** [comment]
Interesting. That's due to a bug in the distro (Mageia) that I'll report upstream.

**Comment by maxice8** [comment]
I think it will happen in any distro that has done the usr-merge, i hit this while packaging for Alpien Linux which didn't did usr-merge yet

**Comment by dfandrich** [comment]
Yes, it will be a problem when run on many distros. We'll make sure the next release doesn't have the issue. in the meantime, "autoreconf -if" will fix it, but it means autoconf needs to be installed.

**Comment by maxice8** [comment]
There are few distros that keep the artificial separation of /usr/bin and /usr/sbin in relation to /bin /sbin but still posix only specifies /bin/sh.

i did `grep -l '^#!/usr/bin/sh' -r . | xargs sed -i 's|^#!/usr/bin/sh|#!/bin/sh|'` in our APKBUILD in the meantime

**Comment by maxice8** [comment]
Also thanks for promptly looking at this issue
